src/ussd_optout_rapidpro.js

```diff
@@ -1,29 +1,333 @@
+                    question: $(question),
+                    error: $(question),
+                    accept_labels: true,
+                    options_per_page: null,
+                    characters_per_page: 160,
+                    choices: [
+                        new Choice("miscarriage", $("Miscarriage")),
+                        new Choice("stillborn", $("Baby was stillborn")),
+                        new Choice("babyloss", $("Baby passed away")),
+                        new Choice("no", $("Msgs aren't helpful")),
+                        new Choice("no", $("Other")),
+                        new Choice("no", $("I prefer not to say"))
+                    ],
+                    next: function(choice) {
+                        self.im.user.set_answer("optout_reason", choice.value);
+                        if(_.includes(["miscarriage", "stillborn", "babyloss"], choice.value)) {
+                            return "state_loss_optout";
+                        } else {
+                            return "state_nonloss_optout";
+                        }  
+                    }  
+                });
+            }
+        });
+
+        self.add("state_trigger_rapidpro_flow", function(name, opts) {
+            var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
+            return self.rapidpro.start_flow(self.im.config.flow_uuid, null, "tel:" + msisdn, {
+                babyloss_subscription: 
+                    self.im.user.get_answer("state_loss_optout") === "yes" ? "TRUE" : "FALSE",
+                delete_info_for_babyloss: 
+                    self.im.user.get_answer("state_delete_info_for_loss") === "yes" ? "TRUE" : "FALSE",
+                delete_info_consent: 
+                        self.im.user.get_answer("state_delete_research_info") === "yes" ? "TRUE" : "FALSE",
+                optout_reason: self.im.user.get_answer("optout_reason"),
+                language: self.im.user.lang,
+                source: "Optout USSD"
+            }).then(function() {
+                if (self.im.user.get_answer("state_delete_info_for_loss") === "yes"){
+                    return self.states.create("state_loss_subscription_without_info_retainment");
+                }else if (self.im.user.get_answer("state_check_previous_optout") === "no"){
+                    return self.states.create("state_nonloss_optout");
+                }else if (self.im.user.get_answer("state_delete_info_for_loss") === "no"){
+                    return self.states.create("state_loss_subscription_with_info_retainment");
+                }else if (self.im.user.get_answer("state_info_deleted") === "yes" ){
+                    return self.states.create("state_nonloss_optout");
+                }else if (self.im.user.get_answer("state_delete_research_info") === "no" ){
+                    return self.states.create("state_nonloss_optout");
+                } else{
+                    return self.states.create("state_nonloss_optout"); 
```

Comment:
It seems like `state_nonloss_optout` is repeated quite a bit, so we can probably just do the checks for `state_loss_subscription_without_info_retainment` and `state_loss_subscription_with_info_retainment`, and if it's neither of those two, just do an
```
} else {
    return self.states.create("state_nonloss_optout"); 
}
```